### PR TITLE
Penguins

### PR DESCRIPTION
**Pull request by LanCartwright**
* Adds Penguin, Penguin baby/chick, and Penguin egg icons.
* Adds Penguin and Penguin baby/chick to the game.
* Adds Penguin and Penguin baby/chick to the holodeck winter.
image

My gift, to you.
 
If anyone is interested in adding Chicken code, such as egg laying and food eating, they are welcome to.

I'm not so experienced in that, and I have some projects I am busy on.

### CODE REVIEW

File: code/modules/mob/living/simple_animal/friendly/farm_animals.dm [hunk 1]
```diff
@@ -297,3 +297,43 @@
 			qdel(src)
 	else
 		STOP_PROCESSING(SSprocessing, src)
+
+// Penguins
+
+/mob/living/simple_animal/penguin
+	name = "\improper baby penguin"
```
Comment (skull132) [inline code/modules/mob/living/simple_animal/friendly/farm_animals.dm]: This should probably just be `penguin`.

**Comment by Pacmandevil** [comment]
Obviously this needs to be a subtype of chicken

File: code/modules/mob/living/simple_animal/friendly/farm_animals.dm (hunk 1 above)
Comment (LanCartwright) [reply code/modules/mob/living/simple_animal/friendly/farm_animals.dm]: major oversight

**Comment by LanCartwright** [comment]
I'll see what I can do with porting this over to development soon.

File: code/modules/mob/living/simple_animal/friendly/farm_animals.dm [hunk 2]
```diff
@@ -298,3 +298,43 @@
...
+/mob/living/simple_animal/penguin
+	name = "\improper King penguin"
+	desc = "A king of the icy regions."
+	icon_state = "penguin"
+	icon_living = "penguin"
+	icon_dead = "penguin_dead"
+	speak = list("Gah Gah!", "NOOT NOOT!", "NOOT!", "Noot", "noot", "Prah!", "Grah!")
+	speak_emote = list("squawks", "gakkers")
+	emote_hear = list("squawk!", "gakkers!", "noots.","NOOTS!")
+	emote_see = list("shakes its beak.", "flaps it's wings.","preens itself.")
```
Comment (Lohikar) [inline code/modules/mob/living/simple_animal/friendly/farm_animals.dm]: Should be `its wings`.

**Pull request merged**